staging/src/k8s.io/cli-runtime/pkg/resource/kustomizevisitor.go

```diff
@@ -38,7 +38,7 @@ type KustomizeVisitor struct {
 // Visit passes the result of a kustomize build to a StreamVisitor.
 func (v *KustomizeVisitor) Visit(fn VisitorFunc) error {
 	kOpts := krusty.MakeDefaultOptions()
-	kOpts.DoLegacyResourceSort = true
+	kOpts.Reorder = krusty.ReorderOptionLegacy
```

Comment:
can we confirm this gives equivalent behavior so kubectl maintains compatibility?